js/templates/modals/settings/moderation.html

```diff
@@ -164,4 +164,10 @@ <h2 class="h3 clrT">Moderation</h2>
       </div>
     </div>
   </div>
+
+  <div class="contentBox padMd clrP clrBr clrSh3">
```

Comment:
In this place I concur with @jjeffryes, if you use the `processingButton` template for this button, then it will get a spinner while a save is in progress. The JS for that is already wired in, so if you just replace:

`<a class="btn clrP clrBAttGrad clrBrDec1 clrTOnEmph js-save"><%= ob.polyT('settings.btnSave') %></a>`

with:

```
      <%= ob.processingButton({
        className: 'btn clrP clrBAttGrad clrBrDec1 clrTOnEmph modalContentCornerBtn js-save',
        btnText: ob.polyT('settings.btnSave')
      }) %>
```

it should work. If it's necessary, you are free to adjust the classes you send into that button via the `className` variable as long as you leave the `js-save` class.